dex/models/_projects/oneinch/oneinch_lop_own_trades.sql

```diff
@@ -19,7 +19,10 @@ select
     , block_number
     , coalesce(dst_token_symbol, '') as token_bought_symbol
     , coalesce(src_token_symbol, '') as token_sold_symbol
-    , array_join(array_sort(array[coalesce(src_token_symbol, ''), coalesce(dst_token_symbol, '')]), '-') as token_pair
+    , case
```

Comment:
@grkhr does this look good to you?

Reply:
as mentioned in the PR I just applied the same logic as in DEX trades. It seems the current logic doesn't work well with mixed-case token symbols - see example: https://dune.com/queries/3864620